services/src/main/java/org/keycloak/services/resources/account/AccountRestService.java

```diff
@@ -481,6 +474,60 @@ public LinkedAccountsResource linkedAccounts() {
         return new LinkedAccountsResource(session, request, client, auth, event, user);
     }
 
+    @Path("/applications")
+    @GET
+    @Produces(MediaType.APPLICATION_JSON)
+    @NoCache
+    public Response applications() {
+        checkAccountApiEnabled();
+        auth.requireOneOf(AccountRoles.MANAGE_ACCOUNT, AccountRoles.VIEW_APPLICATIONS);
+
+        List<ClientModel> clients = new LinkedList<>();
+        List<String> inUseClients = new LinkedList<String>();
+        List<UserSessionModel> sessions = session.sessions().getUserSessions(realm, user);
+        for(UserSessionModel s : sessions) {
+            for (AuthenticatedClientSessionModel a : s.getAuthenticatedClientSessions().values()) {
+                ClientModel client = a.getClient();
+                if(!client.isBearerOnly() && client.getBaseUrl() != null) {
```

Comment:
Just a suggestion for a possible code clarity enhancement (I don't see it as an issue and I'll approve regardless). I think we could remove this condition because it's present here already:
https://github.com/keycloak/keycloak/blob/38f314a2634b829b7edaad6720b014d3dbd395f0/services/src/main/java/org/keycloak/services/resources/account/AccountRestService.java#L522-L524

Reply:
Done